Shared/Samples/Show WFS layer with XML query/ShowWFSLayerWithXMLQueryView.swift

```diff
@@ -0,0 +1,135 @@
+        tableName: .seattleTreesDowntown // The name of the table within the service.
+    )
+    
+    /// Flag to indicate whether data is currently being loaded.
+    @State private var isLoading = false
+    
+    var body: some View {
+        MapViewReader { mapView in
+            MapView(map: map)
+            // Overlay with loading view in the center if data
+            // is currently being loaded.
+                .overlay(alignment: .center) {
+                    if isLoading {
+                        loadingView
+                    }
+                }
+                .task {
+                    do {
+                        // Load the WFS data.
+                        try await loadData()
+                        if let extent = seattleTreesTable.extent, !isLoading {
+                            // Animate the map to the extent of the data over 1 second.
+                            await mapView.setViewpoint(
+                                Viewpoint(boundingGeometry: extent),
+                                duration: 1.0
+                            )
+                        }
+                    } catch {
+                        // If an error occurs during loading, capture it to trigger an alert.
+                        self.error = error
+                    }
+                }
+            // Display an alert if there is an error during data loading.
+                .errorAlert(presentingError: $error)
+        }
+    }
+    
+    /// Asynchronous function to load data from the WFS service
+    func loadData() async throws {
+        isLoading = true
+        // Set the axis order to not swap X and Y (used for coordinate systems.)
+        seattleTreesTable.axisOrder = .noSwap
+        // Use manual cache mode so data must be explicitly loaded from the service.
+        seattleTreesTable.featureRequestMode = .manualCache
+        try await seattleTreesTable.load()
+        // Create a feature layer from the table and add it to the map.
+        let layer = FeatureLayer(featureTable: seattleTreesTable)
+        map.addOperationalLayer(layer)
+        // Populate the feature table with data from the WFS service using an XML query.
+        _ = try await seattleTreesTable.populateFromService(
+            usingXMLRequest: .xmlQuery,
+            clearCache: true
+        )
+        // Mark that loading has completed.
+        isLoading = false
+    }
+    
+    var loadingView: some View {
+        ProgressView(
+            """
+            Loading query
+            data
+            """
+        )
+        .padding()
+        .background(.ultraThinMaterial)
+        .clipShape(.rect(cornerRadius: 10))
+        .shadow(radius: 50)
+        .multilineTextAlignment(.center)
+    }
+}
+
+#Preview {
+    ShowWFSLayerWithXMLQueryView()
+}
+
+private extension URL {
+    /// Static property to return the full URL for the WFS GetCapabilities request.
+    static var wfsUrl: URL {
```

Comment:
1. We typically don't include the type information in a variable name unless necessary
2. The name can be `seattleDowntownFeatures`